src/Makefile.vc6

```diff
@@ -126,7 +126,7 @@ $(WSOCK_DLL) $(WSOCK_LIB): $(WSOCK_DEF) $(OBJECTS) $(NON_EXPORT_OBJ) compile_jua
 	@type link.tmp >> $(WSOCK_MAP)
 	@del $(WSOCK_LIB:.lib=.exp)
 
-install:
+install: all
```

Comment:
That's okay. But now that `install` is not part of `all`, I'll removed the check for `"$(USER)" != "AppVeyor"`.